apps/journeys-admin/src/components/NewPageWrapper/MainPanelBody/MainPanelBody.tsx

```diff
@@ -5,11 +5,13 @@ import { usePageWrapperStyles } from '../utils/usePageWrapperStyles'
 export interface MainPanelBodyProps {
   children: ReactNode
   bottomPanelChildren?: ReactNode
+  isEdit?: boolean
 }
 
 export function MainPanelBody({
   children,
-  bottomPanelChildren
+  bottomPanelChildren,
+  isEdit
```

Comment:
I've also ran into cases where the padding needs to be removed. Can we refactor this so it can be more widely used?
```suggestion
  disableGutters = false
```